api/diagnostics.go

```diff
@@ -22,6 +22,10 @@ import (
 	"github.com/dcos/dcos-diagnostics/util"
 	"github.com/dcos/dcos-go/exec"
 	"github.com/shirou/gopsutil/disk"
+	"regexp"
+	"google.golang.org/genproto/googleapis/type/date"
+	"strconv"
+	"sort"
```

Comment:
Please sort imports. You can use [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports)

```diff
api/diagnostics.go
diff -u api/diagnostics.go.orig api/diagnostics.go
--- api/diagnostics.go.orig	2018-12-29 01:24:24.664985289 +0000
+++ api/diagnostics.go	2018-12-29 01:24:24.664985289 +0000
@@ -22,10 +22,10 @@
 	"github.com/dcos/dcos-diagnostics/util"
 	"github.com/dcos/dcos-go/exec"
 	"github.com/shirou/gopsutil/disk"
-	"regexp"
 	"google.golang.org/genproto/googleapis/type/date"
-	"strconv"
+	"regexp"
 	"sort"
+	"strconv"
 )
```